Qt6 update

acpaquette opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

acpaquette committed:
Initial updates for qt6

acpaquette committed:
Add skip for ppc64le

acpaquette committed:
Bump build number

acpaquette commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ Selectors are suggested to take a ``<two spaces>#<one space>[<expression>]`` form. See lines [17]

For **recipe/meta.yaml**:

 * ℹ️ License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.

Documentation on acceptable licenses can be found [here]( https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions ).
 * ℹ️ The feedstock is lowering the image versions for one or more platforms: {'linux_64': 'cos7'} (the default is alma9). Unless you are in the very rare case of repackaging binary artifacts, consider removing these overrides from conda-forge.yml in the top feedstock directory.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/17776112851](https://github.com/conda-forge/conda-forge-webservices/actions/runs/17776112851). Examine the logs at this URL for more detail.</sub>

acpaquette committed:
Appease the linter

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.

Documentation on acceptable licenses can be found [here]( https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions ).
 * ℹ️ The feedstock is lowering the image versions for one or more platforms: {'linux_64': 'cos7'} (the default is alma9). Unless you are in the very rare case of repackaging binary artifacts, consider removing these overrides from conda-forge.yml in the top feedstock directory.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/17776189724](https://github.com/conda-forge/conda-forge-webservices/actions/runs/17776189724). Examine the logs at this URL for more detail.</sub>

acpaquette committed:
Add small build fixes

acpaquette committed:
Update linux 64 version

acpaquette committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.16.07.18.04

acpaquette committed:
Updated libgl deps

acpaquette committed:
Add qt6-main in build section for osx-arm64

acpaquette committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.16.07.18.04

acpaquette committed:
Remove libgl1-mesa-glx

acpaquette committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.16.07.18.04

acpaquette committed:
Added mesa-libGL-devel back

acpaquette committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.16.07.18.04

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6

acpaquette committed:
Add qt6-main and mesa-libgl-devel to build section

jschueller commented:
I dont remember, sorry

acpaquette committed:
Added libgl-devel and qt6-main to build section of recipe

acpaquette committed:
Added unix flag to qt6-main in build section

acpaquette committed:
Changed flags to linux on libgl-devel and qt6-main in build section

acpaquette committed:
Changed flag to unix for qt6-main in build section

acpaquette commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/17841147846](https://github.com/conda-forge/conda-forge-webservices/actions/runs/17841147846). Examine the logs at this URL for more detail.</sub>

acpaquette committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.09.17.39.06

Other tools:
- conda-build 25.11.0
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.10

acpaquette committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.09.17.39.06

Other tools:
- conda-build 25.11.0
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.10

acpaquette committed:
Add back linking step

acpaquette committed:
Reduced requirement set

acpaquette committed:
Add patch to pickup conda environment include directory when building

acpaquette committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.11.16.09.59

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6

acpaquette committed:
Clean up recipe

acpaquette committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.11.16.09.59

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.10

acpaquette committed:
Add libgl to host section

acpaquette commented:
Closing this in favor of #46, it would be ideal if the linux arm64 builds could be enabled again. I'm not entirely sure why they were disabled
